_posts/en/newsletters/2021-03-24-newsletter.md

```diff
@@ -0,0 +1,342 @@
+      create an incentivized early warning system.
+
+    - *We could add a hash-style address after taproot is activated:* if
+      a significant number of users really do think the sudden
+      appearance of a powerful QC is a threat, we could use a follow-up
+      soft fork to [add][dashjr quantum] an alternative P2PKH-style taproot
+      address type that uses hashes.  However, this has consequences
+      that caused several respondents to oppose it:
+
+        - It'll create confusion
+
+        - It'll use more block space
+
+        - It [reduces][poelstra anon] the size of taproot's anonymity
+          set, both directly and when protocols such as [ring signature
+          membership proofs][nick ring] or [Provisions][] are being used
+
+    - *Bandwidth/storage costs versus CPU costs:* it's [possible][rubin
+      recovery] to eliminate the extra 32-byte storage overhead from
+      hash indirection by deriving the public key from a signature and
+      the transaction data it signs, a technique called *key recovery*.
+      Again, this was opposed.  Key recovery requires a [significant amount of CPU][corallo
+      recovery overhead] that would slow down nodes and also prevents
+      the use of schnorr batch validation that can make historic block
+      verification up to three times faster.  It also [makes][poelstra
+      slowdowns] anonymous membership proofs and related techniques both
+      harder to develop and much more CPU intensive.  There
+      [may][poelstra patent story] also be a [patent][US7215773B1] issue.
+
+  As of this writing, it appears the mailing list discussion has
+  concluded without any obvious loss of community support for taproot.
+  As researchers and businesses continue improving the state of the art
+  in quantum computing, we expect to see future discussions about how to
+  best keep Bitcoin secure.
+
+- **Weekly taproot activation meetings:** ten weekly meetings to discuss
+  details related to activating [taproot][topic taproot] have been
+  scheduled for each Tuesday at 19:00 UTC in the
+  [##taproot-activation][] IRC channel, with the first
+  [meeting][activation meeting log] occurring yesterday (March 23rd).
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **OKCoin launches Lightning deposits and withdrawals:**
+  A [blog post][okcoin lightning blog] outlines OKCoin's Lightning deposit and withdrawal
+  support. They also lowered their minimum deposit/withdrawal limit from 0.001 to
+  0.000001 BTC as a result. At this time, 0.05 BTC is OKCoin's limit when transacting
+  using LN.
+
+- **BitMEX announces bech32 support:**
+  In a [blog post][bitmex bech32 blog], BitMEX detailed the launch plans for
+  bech32 deposit support. BitMEX had [previously][news77 bitmex
+  bech32 send] rolled out [bech32][topic bech32] withdrawal (send) support.
```

Comment:
It feels a little bit strange to link to the bech32 topic from the second instance of "bech32" in this sentence rather than the first.